_topics/en/miniscript.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Miniscript
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Scripts and Addresses
+  - Wallet Collaboration Tools
+  - Developer Tools
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Miniscript** allows software to automatically analyze a script,
+  including determining what data is necessary to create a witness that
+  fulfills the script and allows any bitcoins protected by the script to
+  be spent.  With miniscript telling the wallet what it needs to do,
+  wallet developers don't need to write new code when they switch from
+  one script template to another.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  This automation for the large range of scripts supported by miniscript
+  allows wallets to be much more dynamic about the scripts they use,
```

Comment:
Perhaps something like "Miniscript's structured representation of a large range of Bitcoin scripts allows..."